roles/kafka_broker/tasks/kraft_rbac_mtls.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Add Super Users list to Kafka Controller Properties
+  set_fact:
+    kafka_controller_final_properties: "{{ hostvars[controller_host]['kafka_controller_final_properties'] | combine(
+      {
+          'super.users': hostvars[controller_host]['super_users'] + ';' + hostvars[groups.kafka_broker[0]]['kafka_broker_principal']
+      }
+    ) }}"
+
+- name: Create Kafka Controller Config
```

Comment:
If the logic for create_config.yml is added here rather than as a separate file in kafka_controller, would that cause any issue?
I'm just thinking whether the create_config.yml has a separate utility in itself beyond being called from here.

Reply:
yes, create_config needs the template from kafka controller role, so it needs to be present inside kafka_controller role to call it